examples/simple_hpc_pbs_zfs/readme.md

```diff
@@ -0,0 +1,66 @@
+# Build a PBS compute cluster
+![Build Status](https://azurecat.visualstudio.com/hpccat/_apis/build/status/azhpc/examples/simple_hpc_pbs?branchName=master)
+
+Visualisation: [config.json](https://azurehpc.azureedge.net/?o=https://raw.githubusercontent.com/Azure/azurehpc/master/examples/simple_hpc_pbs/config.json)
+
+This example will create an HPC cluster ready to run with PBS Pro.  The headnode will export a ZFS filesystem for the nodes to use.
+
+## Initialise the project
+
+To start you need to copy this directory and update the `config.json`.  Azurehpc provides the `azhpc-init` command that can help here by compying the directory and substituting the unset variables.  First run with the `-s` parameter to see which variables need to be set:
+
+```
+azhpc-init -c $azhpc_dir/examples/simple_hpc_pbs -d simple_hpc_pbs -s

```

Comment:
Please fix the dir name as well as below

Reply:
fixed

Reply:
done